Documentation/image-fetching-behavior.md

```diff
@@ -0,0 +1,26 @@
+# Image fetching behavior
+
+When fetching, rkt will try to avoid unnecessary network transfers: if an updated image is already in the local store there's no need to download it again.
+
+This behavior can be controlled with the `--store-only` and `--no-store` flags.
+
+## General behavior
+
+--store-only | --no-store |  behavior          | description
+ ----------- | ---------- | ------------------ | -------------------------------------------------------------------------------
+1            | 0          | store              | Just check the store
+0            | 1          | remote             | Do remote download (handling caching logic for http(s):// and docker://)
+0            | 0          | store then remote  | Default Behavior. Do `store` and if it doesn't return an image do `remote`
+
+## Details
+
+behavior     | image              | detailed behavior
+------------ | ------------------ | --------------------------------------------------------------------------------------------
+store        | file://            | Just use the file.
+store        | http(s)://         | Search in the store remote table if the URL is available and use the saved image.
```

Comment:
I'll use "Check for the URL in the local store. If found, use the corresponding image."